pkg/action/executor/move_util.go

```diff
@@ -525,6 +530,8 @@ func createClonePod(client *kclient.Clientset, pod *api.Pod, parent *unstructure
 	npod.Annotations = annotations
 	// this annotation can be used for future garbage collection if action is interrupted
 	util.AddAnnotation(npod, TurboActionAnnotationKey, TurboMoveAnnotationValue)
+	// This label is used for garbage collection if a given action leaks this pod.
+	util.AddLabel(npod, TurboGCLabelKey, TurboGCLabelVal)
```

Comment:
I'm probably misunderstanding the strategy; shouldn't we also have some code too to remove the label so that properly moved pods will not be cleaned up?

Reply:
We do update the labels on the cloned pod as the last step in the move function.
Ref (https://github.com/turbonomic/kubeturbo/pull/520/files#diff-db23cf9770312fe51ba93b7df47be72f93f5e1df923e4c2c1e10c68e4b0439f0R244)